README.md

```
@@ -1,96 +1,278 @@
-### 3. Setting up Upstash Redis
-
-Follow the guide below to set up Upstash Redis. Create a database and obtain `UPSTASH_REDIS_REST_URL` and `UPSTASH_REDIS_REST_TOKEN`. Refer to the [Upstash guide](https://upstash.com/blog/rag-chatbot-upstash#setting-up-upstash-redis) for instructions on how to proceed.
+### Step 3: Run Database Migrations
+Generate and apply migrations to your local/target PostgreSQL database using Drizzle:
+```bash
+bun x drizzle-kit generate
+bun run db:migrate
+```
 
-### 4. Fill out secrets
+### Step 4: AlphaEarth Satellite Index Setup (Optional)
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)